src/main/java/com/google/sps/data/ActivityEvent.java

```diff
@@ -2,16 +2,38 @@
 
 import java.util.List;
 
-public class ActivityEvent {
+public final class ActivityEvent {
   private String userID;
-  private long timestamp;
+  private long startTimestamp;
+  private long endTimestamp;
   private Activity activity;
   private List<String> guests;
 
-  public ActivityEvent(String userID, long timestamp, Activity activity, List<String> guests) {
+  public ActivityEvent(String userID, long startTimestamp, long endTimestamp, Activity activity, List<String> guests) {
     this.userID = userID;
-    this.timestamp = timestamp;
+    this.startTimestamp = startTimestamp;
+    this.endTimestamp = endTimestamp;
     this.activity = activity;
     this.guests = guests;
   }
+
+  public String getUserId() {
+    return userID;
+  }
+
+  public long getStartTimestamp() {
+    return startTimestamp;
+  }
+
+  public long getEndTimestamp() {
+    return endTimestamp;
+  }
+
+  public Activity getActivity() {
```

Comment:
All these functions that return can have some javadoc that tells us what is being returned and what that's used for. Similar to how TimeRange was made in week5-tdd.